databases/models/ChannelMessages.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class ChannelMessages extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  ChannelMessages.init(
+    {
+        id: {
+            type: DataTypes.INTEGER,
+            allowNull: false,
+            primaryKey: true,
+            autoIncrement: true
+        },
+        message_id: {
+            type: DataTypes.UUID,
+            allowNull: false
+        },
+        channel_id:{
+            type: DataTypes.UUID,
+            allowNull: false,
+        },
+        user_id: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        message: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        is_read_by_admin: {
+            type: DataTypes.BOOLEAN,
+            defaultValue: false
+        },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `is_read_by_admin` field has a default value of `false`. This is good practice for boolean fields where a default value makes sense.



<!-- This is an auto-generated comment by OSS CodeRabbit -->